src/webui/www/private/views/rssDownloader.html

```diff
@@ -742,12 +745,14 @@
                     $("lastMatchText").textContent = "QBT_TR(Last Match: Unknown)QBT_TR[CONTEXT=AutomatedRssDownloader]";
                 }
 
-                if (rulesList[ruleName].torrentParams.stopped === null)
+                // use `== null` to check for both `null` and `undefined`
+                if (rulesList[ruleName].torrentParams.stopped == null)
```

Comment:
As per https://stackoverflow.com/questions/2559318/how-to-check-for-an-undefined-or-null-variable-in-javascript:
>In general it is recommended to use `===` instead of `==`. The proposed solution is an exception to this rule. The [JSHint syntax checker](http://www.jshint.com/docs/#options) even provides the `eqnull` option for this reason.
>
>From the [jQuery style guide](http://contribute.jquery.org/style-guide/js/):
>
>>Strict equality checks (===) should be used in favor of ==. The only exception is when checking for undefined and null by way of null.

So I turned on the smart mode of ESLint `eqeqeq` rule so that it allows for such a comparison.

Reply:
Still I would like to enforce strict comparison operators universally, it keeps the code uniform. It is verbose in these code but we only have a few  instances for it (2 so far).

Reply:
Also I consider verbose in this case is a plus. It become clear that it is expecting `null` and `undefined` values.

Reply:
> Also I consider verbose in this case is a plus. It become clear that it is expecting `null` and `undefined` values.

Ok. I had a feeling about it.
I just feel difficulty understanding JS. Is the following correct?
```js
if ((rulesList[ruleName].torrentParams.stopped === undefined) || (rulesList[ruleName].torrentParams.stopped === null))
```
I noticed that someone describes check for `undefined` as the following:
```js
typeof(some_variable) === "undefined"
```

Reply:
I agree that there are benefits to being verbose, but have also found this smart rule to be an improvement to convenience on other projects with much stricter codebases.

Reply:
Anyway, it still requires verbosity either in the code or in the comment (so that someone does not break it later by replacing `==` with `===`).

Reply:
You avoid this by enforcing the general rule that all checks against null or undefined use `==`.

Reply:
> > Also I consider verbose in this case is a plus. It become clear that it is expecting `null` and `undefined` values.
> 
> Ok. I had a feeling about it. I just feel difficulty understanding JS. Is the following correct?
> 
> ```js
> if ((rulesList[ruleName].torrentParams.stopped === undefined) || (rulesList[ruleName].torrentParams.stopped === null))
> ```
> 
> I noticed that someone describes check for `undefined` as the following:
> 
> ```js
> typeof(some_variable) === "undefined"
> ```

It depends. If `some_variable` is declared but hasn't assigned a value then `some_variable === undefined` will hold. If `some_variable` wasn't declared then `some_variable === undefined` will throw exception and `typeof(some_variable) === "undefined"` must be used.